selenium_tests/lti_emailer/emailer_main_page_tests.py

```diff
@@ -0,0 +1,16 @@
+import unittest
+
+from selenium_tests.lti_emailer.emailer_base_test_case import EmailerBaseTestCase
+from selenium_tests.lti_emailer.page_objects.emailer_mainpage_page_object import EmailerMainPage
+
+
+class EmailerMainPageTests(EmailerBaseTestCase):
+
+    def test_tool_is_loaded_on_page(self):
+        """Check that page is loaded by checking against site title"""
+        driver = self.driver
+        page = EmailerMainPage(driver)  # instantiate
+        page.is_loaded()
```

Comment:
This isn't asserting anything...? I think it needs to be wrapped in `self.assertTrue(...)`, or else this test will always pass.